### PR TITLE
Add initial draft of admin UI Version Info guide (#284)

### PR DESCRIPTION
**Pull request by kylesalk**
Documentation based on what I saw in the dashboard. I can elaborate on any portion if needed.

Just a note: I followed the other admin-ui contributions and put the image in the top-level /img directory, but I think /_docs/img would make more sense.

### CODE REVIEW

**Comment by zoltanbedi** [comment]
Hi @kylesalk awsome work! Can you please change the casing of sensenet in the docs to all lowercase?

File: _docs/admin-ui/version-info.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+source_url: "https://github.com/SenseNet/sensenet.github.io/blob/master/docs/admin-ui/version-info.md"
+category: Admin UI
+version: v7.0
+tags: [admin ui, version, settings]
+description: An overview the version information available for the running instance
+---
+
+# Verison Info
+
+Version Info provides information on the currently running instance of SenseNet.
```
Comment (herflis) [inline _docs/admin-ui/version-info.md]:
```suggestion
Version Info provides information on the currently running instance of sensenet.
```

File: _docs/admin-ui/version-info.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+![Version Info](/img/setup-dashboard.png "Version information page")
+
+## Admin UI
+
+The Admin UI section contains information regarding the SenseNet client.
```
Comment (herflis) [inline _docs/admin-ui/version-info.md]:
```suggestion
The Admin UI section contains information regarding the sensenet client.
```

File: _docs/admin-ui/version-info.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+**Application version**: Running version of the SenseNet client.
```
Comment (herflis) [inline _docs/admin-ui/version-info.md]:
```suggestion
**Application version**: Running version of the sensenet client.
```

File: _docs/admin-ui/version-info.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+**Branch name**: SenseNet client Github branch in use. Clicking on the branch will
```
Comment (herflis) [inline _docs/admin-ui/version-info.md]:
```suggestion
**Branch name**: sensenet client Github branch in use. Clicking on the branch will
```

File: _docs/admin-ui/version-info.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+direct users to the Github client repository.
+
+**Commit Hash**: Latest commit for the current branch. Clicking on the hash
+will direct users to the file diff for the commit.
+
+## Components
+
+Several SenseNet components are utilized with the SenseNet installation. This
```
Comment (herflis) [inline _docs/admin-ui/version-info.md]:
```suggestion
Several sensenet components are utilized with the sensenet installation. This
```

File: _docs/admin-ui/version-info.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+section describes the components in use and the respective versions.
+
+## Installed packages
+
+Similar to the Components section, the installed packages detail all SenseNet
```
Comment (herflis) [inline _docs/admin-ui/version-info.md]:
```suggestion
Similar to the Components section, the installed packages detail all sensenet
```

File: _docs/admin-ui/version-info.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+packages installed, versions, and package descriptions for the installed
+version of SenseNet.
```
Comment (herflis) [inline _docs/admin-ui/version-info.md]:
```suggestion
version of sensenet.
```

File: _docs/admin-ui/version-info.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+## Assemblies
+
+When SenseNet is compiled, additional assemblies are linked for use. These
```
Comment (herflis) [inline _docs/admin-ui/version-info.md]:
```suggestion
When sensenet is compiled, additional assemblies are linked for use. These
```

File: _docs/admin-ui/version-info.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+assemblies can be standard .NET libraries, open-source libraries (e.g. nuget),
+or locally built libraries (e.g. SenseNet).
```
Comment (herflis) [inline _docs/admin-ui/version-info.md]:
```suggestion
or locally built libraries (e.g. sensenet).
```

**Pull request merged**